test/clj/rems/test_browser.clj

```diff
@@ -299,6 +303,16 @@
         ;; pick two options for the multi-select field:
         (btu/check-box "Option2")
         (btu/check-box "Option3")
+        ;; fill in two rows for the table
+        (btu/scroll-and-click (keyword (str table-field-id "-add-row")))
+        (btu/wait-visible (keyword (str table-field-id "-row0-col1")))
+        (btu/scroll-and-click (keyword (str table-field-id "-add-row")))
+        (btu/wait-visible (keyword (str table-field-id "-row1-col1")))
+        (btu/fill-human (keyword (str table-field-id "-row0-col1")) "a")
+        (btu/fill-human (keyword (str table-field-id "-row0-col2")) "b")
+        (btu/fill-human (keyword (str table-field-id "-row1-col1")) "c")
+        (btu/fill-human (keyword (str table-field-id "-row1-col2")) "d")
+
```

Comment:
Should we add one test of the table rendering on browser side too? This just checks the data entry side?

Reply:
Good idea.